File: issta2018artifacts/minimized-bugs/fixed-key/Exdent.java
// version of code that causes bug fixed by plume-lib rev. 339602f

public class Exdent {
    boolean enabled = true;
    String indent_str;
    final String INDENT_STR_ONE_LEVEL = "    ";

    /** Exdents: reduces indentation and pops a start time. */
        /*@ public normal_behavior
@ requires indent_str.length() > INDENT_STR_ONE_LEVEL.length();
@ ensures true;
@*/
    public void exdent() {
	//:: error: (argument.type.incompatible)
	indent_str = indent_str.substring(0, indent_str.length() - INDENT_STR_ONE_LEVEL.length());
    }
}

File: issta2018artifacts/minimized-bugs/buggy-clousot/PlumeLib/Exdent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

// version of code that causes bug fixed by plume-lib rev. 339602f

public class Exdent
{
    bool enabled = true;
    string indent_str;
    readonly string INDENT_STR_ONE_LEVEL = "    ";

    /** Exdents: reduces indentation and pops a start time. */
    public void exdent()
    {
        //:: error: (argument.type.incompatible)
        indent_str = indent_str.Substring(0, indent_str.Length - INDENT_STR_ONE_LEVEL.Length);
    }
}